src/scripts.js

```diff
@@ -66,8 +66,18 @@ function dropYear(dates) {
   return reformattedDates
 }
 $(document).ready(function () {
-
+console.log('ready')
```

Comment:
Since we are using webpack we do not need the code on line 68 - the local server and importing $ from jquery at the top of the file handles all of the jquery making the document.ready unnecessary - same for the console.log - Brittany mentioned this yesterday, and it sounds like it is best practice to keep the console.logs on your local side and don't push them up to master.